kimchi/src/prover.rs

```diff
@@ -1184,9 +1235,13 @@ pub mod caml {
         pub evals: (CamlProofEvaluations<CamlF>, CamlProofEvaluations<CamlF>),
         pub ft_eval1: CamlF,
         pub public: Vec<CamlF>,
-        pub prev_challenges: Vec<(Vec<CamlF>, CamlPolyComm<CamlG>)>,
+        pub prev_challenges: Vec<CamlChallenge<CamlG, CamlF>>, //Vec<(Vec<CamlF>, CamlPolyComm<CamlG>)>,
     }
 
+    //
+    // CamlProverCommitments<CamlG>
```

Comment:
I think this comment should be more descriptive, giving new info.

Reply:
I think these are just "separators" whenever we define a new Caml struct